package.json

```diff
@@ -8,8 +8,9 @@
   },
   "scripts": {
     "analyze": "NODE_ENV=production webpack --config config/prod.webpack.config.js --env analyze=true",
-    "build": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
-    "deploy": "npm-run-all build lint test",
+    "build:prod": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
+    "build:staging": "NODE_ENV=staging webpack --config config/prod.webpack.config.js",
+    "deploy": "npm-run-all build:prod lint test",
```

Comment:
Do we still need this `deploy` script target?

Reply:
Yea we can remove it. I can do that in a separate PR too